config/settings.yml

```diff
@@ -1048,127 +1048,143 @@
   :authentication_timeout: 30.seconds
   :url:
 :workers:
+  :cockpit_ws_worker:
+    :count: 1
+    :inherits_from: worker_base
+  :ems_refresh_core_worker:
```

Comment:
This should have been removed already, we don't have a refresh_core_worker anymore

Reply:
Less is More!

Reply:
Cool...I'll send up a separate PR

Reply:
Oh I already did 🤦‍♂️ https://github.com/ManageIQ/manageiq/pull/20854

This is just a bug in my reorganization.

Reply:
No its already gone haha